plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -10,6 +10,7 @@ class TemplateFiller
 
     def initialize(release_doc, template, index:)
       @doc = release_doc
+      @deploy_group = DeployGroup.with_deleted { @doc.deploy_group }
```

Comment:
we should not do this, a deleted deploy group does not need new deployments ... only view related code needs to be addressed